packages/atomic/src/components.d.ts

```diff
@@ -393,7 +393,7 @@ export namespace Components {
          */
         "iconAssetsPath": string;
         /**
-          * Initializes the connection with the headless commerce engine using options for accessToken (required), organizationId (required), renewAccessToken, organizationEndpoints (recommended), and platformUrl (deprecated).
+          * Initializes the connection with the headless commerce engine using options for accessToken (required), organizationId (required), organizationEndpoints (required), and renewAccessToken.
```

Comment:
I would sprinkle this article: https://docs.coveo.com/en/mcc80216/manage-an-organization/organization-endpoints throughout the various JSDoc comment to showcase how to configure it